CHANGELOG.md

```diff
@@ -131,6 +131,10 @@ The format is based on [Keep a Changelog].
   an error would be thrown ([#152]).
 * When completing filenames and a match is required, non-normalized
   paths (e.g., `~/Documents//etc/hosts`) are accepted ([#190]).
+* Pressing TAB when nothing matches shows a “No match” message in the
+  minibuffer instead of signaling an error and erasing the minibuffer
+  contents ([#193]).  If ‘completion-fail-discreetly’ is non-nil,
```

Comment:
This style of quotes won't render correctly in Markdown, so I updated master to use regular backticks on both sides.

Reply:
Thank you, I did not notice those quotes.